objects/DentateGyrus.hoc

```diff
@@ -259,10 +259,11 @@ begintemplate DentateGyrus
         PP_input2MC_  	= 0         // stimulate MC with PP
         PP_rate_        = 10.       // rate of PP Poisson input
         PP_rate_noise_  = 0.        // rate of PP Poisson noise
-        PP_box_nr_      = 6         // number of active PPs
-        PP_box_stop_	= 35.       // time of box [ms]
-        PP_box_start_	= 5.        // shift of box [ms]
-        PP_box_nspk_    = 3         // number of spike per active PP
+        PP_box_nr_      = 10         // number of active PPs            TODO: need to be able to scale prop. to. desired freq (SS)
+        PP_box_stop_	= 1000.       // time of box [ms]
+        PP_box_start_	= 0.        // shift of box [ms]
+        PP_box_nspk_    = 25        // number of spike per active PP    TODO: need to be able to scale prop. to. desired freq (SS)
+        PP_freq_        = 3         // oscillation frequency
```

Comment:
what are the units here, hz? how is it being changed for different PP oscillation frequencies?